pkg/virt-controller/watch/vmi.go

```diff
@@ -2100,3 +2111,36 @@ func (c *VMIController) getVolumePhaseMessageReason(volume *virtv1.Volume, names
 	}
 	return virtv1.VolumePending, PVCNotReadyReason, "PVC is in phase Lost"
 }
+
+func (c *VMIController) handleDynamicInterfaceRequests(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error {
+	log.Log.Object(pod).V(4).Info("network iface hotplug for pod")
```

Comment:
Every second line there is a debug log here.
I think there is a need to reduce it to one, adding there all that you need.

Reply:
Removed some.

Is it better now  ?